pages/common/routers/datatable.js

```diff
@@ -153,6 +153,15 @@ module.exports = ({
     const stringifier = csv({
       bom: true,
       header: true,
+      cast: {
+        string: (value) => {
+          if (value && /[0-9]+/.test(value) && !value.includes('"')) {
+            return { value: `="${value}"`, quote: false };
+          } else {
+            return { value, quote: false };
```

Comment:
What is `quote: false` doing here?

Reply:
Reading the docs, just reverting to the default seems safest.
```suggestion
            return value;
```